packages/patterns-react/src/patterns/LeadSpace/__stories__/LeadSpace.stories.js

```diff
@@ -33,16 +34,53 @@ if (DDS_LEADSPACE) {
         productive: 'productive',
       };
 
+      /**
+       * Return icon component based on the parameter
+       *
+       * @param {string} icon icon name parameter
+       * @returns {*} Icon JSX component or null
+       */
+      const selectIcon = icon => {
```

Comment:
can you follow what we have for LeadSpaceCentered.stories.js to keep consistency?
```
const iconMap = {
        ArrowRight20,
        ArrowDown20,
        Pdf20,
      };

      const icons = {
        ArrowRight: 'ArrowRight20',
        ArrowDown: 'ArrowDown20',
        Pdf: 'Pdf20',
        none: null,
      };

      const buttons = [
        {
          link: '',
          copy: 'Primary action button',
          renderIcon:
            iconMap[select('primary button icon', icons, icons.ArrowRight)],
        },
        {
          link: '',
          copy: 'Secondary',
          renderIcon:
            iconMap[select('secondary button icon', icons, icons.ArrowRight)],
        },
      ];
```

Reply:
Sure, I'll apply this logic, thanks for sharing it